packages/adapter/src/lib/adapter/adapter.ts

```diff
@@ -3428,6 +3428,60 @@ export class AdapterClass extends EventEmitter {
         }
     }
 
+    // external signature
+    objectExists(id: string, options?: Record<string, any>): Promise<boolean> | void;
+
+    /**
+     * Checks if an object exists to the given id, id will be fixed first
+     *
+     * @param id id of the object
+     * @param options optional user context
+     */
+    objectExists(id: unknown, options: unknown): Promise<boolean> | void {
+        if (!adapterObjects) {
+            this._logger.info(`${this.namespaceLog} objectExists not processed because Objects database not connected`);
+            return;
```

Comment:
why we do not return the DB CLOSED error as in all other methods?

Reply:
it is not reuturned, maybeCb checks for the error and if this is the case it just resolves the Promise with nothing `Promise<void`.

https://github.com/ioBroker/ioBroker.js-controller/blob/27ca332aa6db589b0fd6e3df1b363e837a31d470/packages/common/src/lib/common/maybeCallback.ts#L145